### DISCUSSION TITLE
My Copilot subscription ended, but I'm still student

### DISCUSSION THREAD

**Discussion by koteelok** [community/community#43650] [GitHub Education]
Hello, I am a participant in the Github Students Pack program, but recently I was denied access to Copilot, which should be available to me as a verified student.

How can I fix that?

UPD: I got an email and my subscription was renewed 2-3 days ago, but suddenly it stopped working again.

**Comment by mamba-byte** [comment]
same

**Comment by lhr000lhr** [comment]
same, I am an AKL uni student.

**Comment by monstyy** [comment]
Same here, and I'm a CU uni student.

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by TiagoRibeiro25** [comment]
Same here (it was working few minutes ago)

**Comment by so1ve** [comment]
same

**Comment by erdoganabaci** [comment]
same :(

**Comment by luuknnh** [comment]
Same, was working earlier today, now it doesn't work anymore :(

**Comment by ajnjanssen** [comment]
Same, cant afford it, fml

**Reply by ajnjanssen** [reply]
Mine works again. Took a day and i only logged out and log back in inside vscode

**Comment by dammyammy** [comment]
same here

**Comment by devhysterical** [comment]
same

**Comment by dikaewiwurae7** [comment]
same

**Comment by b14ck0ps** [comment]
same problem

**Comment by tubarao312** [comment]
Just got the same issue. Apparently, everyone had received an email regarding it ending on 9/1/2023 a few months back. Let's see what happens..

**Comment by 0xlesen** [comment]
It's all right now, I've received the mail. "Thank you for renewing your free access to GitHub Copilot. We hope you enjoy using GitHub Copilot and participating in the developer community!Your access to GitHub Copilot will expire on 2023-02-07."

**Reply by DannisNguyen12** [reply]
Hi, 

How could you renew it? Can you guide me. I got copilot stopped all suddenly today.

**Comment by zeroscc** [comment]
same

**Comment by cmd2001** [comment]
same

**Comment by pers0na2dev** [comment]
same

**Comment by racns** [comment]
same

**Comment by otkrickey** [comment]
Check: #43634

**Comment by lleviathe** [comment]
same here

**Comment by ChiragAgg5k** [comment]
same...

**Reply by ChiragAgg5k** [reply]
<img width="605" alt="Screenshot 2023-01-09 at 4 48 02 PM" src="https://user-images.githubusercontent.com/110609663/211296477-0aaeaba0-d6e4-41ec-a0bc-0752568b1dc6.png">

got the renewal.

**Comment by ygtdmn** [comment]
same here

**Comment by MelenkoOleksandr** [comment]
Same problem. I'm a student with approval on Git Education and I have Student Developer Pack but Copilot not working...
How long do I need to wait to get access again or where I can reapply for the Copilot subscription?
Also I can't reverify my academic affiliation because I already have a submitted student account

![image](https://user-images.githubusercontent.com/60272414/211310391-5cda7516-dc20-45cd-b287-489a7489dfdf.png)
![image](https://user-images.githubusercontent.com/60272414/211309650-a70571a2-52ad-4cd4-9f77-88a5dc2444f0.png)
![image](https://user-images.githubusercontent.com/60272414/211309723-7311f608-f12d-4639-a9aa-96da6a8ddfdc.png)

**Reply by alainjr10** [reply]
This is the same issue i have

**Comment by thuongtruong1009** [comment]
> Hello, I am a participant in the Github Students Pack program, but recently I was denied access to Copilot, which should be available to me as a verified student.
> 
> How can I fix that?

Don't worry. This is the notification from Copilot and your account will be extended until your student email expires. I've had a case like this but it's not a serious problem. Don't worry, logout and re-enter your Github account on VS Code and it's working fine

**Comment by TeslaSesla** [comment]
same =(

**Comment by cnahmetcn** [comment]
same :(

**Reply by cnahmetcn** [reply]
e-mail arrived. my problem is solved

**Comment by mubashiriqbal97** [comment]
same

**Comment by matt-gribben** [comment]
I have this same issues, recived an email on December 11th saying GitHub Copilot will expire on 2023-01-10 which I guess it has done, but why? I'm still on an approved student account and still have github pro etc?

**Comment by Yawarzy** [comment]
I received the email about the renewal. But the Copilot tab in the GitHub settings still shows expired.

**Comment by siddhant1999** [comment]
Same

**Comment by camesaro** [comment]
Igual

**Comment by Nikhil8557** [comment]
If you are a student you will get Github Pro for free

**Comment by pChitral** [comment]
same
